src/game_api/vtable_hook.hpp

```diff
@@ -55,7 +55,7 @@ struct VTableDetour<RetT(ClassT*, ArgsT...), Index>
                 s_Functions[self](self, args..., s_Originals[vtable]);
                 return;
             }
-            s_Originals[vtable](self, std::move(args)...);
+            s_Originals[vtable](self, args...);
```

Comment:
This is kind of bad, but also all the parameters in virtuals are just simple values and pointers, so there shouldn't be any big optimization issues.
Had to do this for reference parameters that won't work with std::move

Reply:
Reverted this for now, it's kind of nice to have parameters that use reference and not pointer, to show that game does not do null check for it, but i guess it doesn't matter much since it is not shown in the API doc